broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java

```diff
@@ -108,26 +113,26 @@ private boolean checkConfig() {
 
     private void init() {
         if (!checkConfig()) {
-            LOGGER.error("check broker metrics config failed, will not export metrics");
+            LOGGER.error("check metrics config failed, will not export metrics");
             return;
         }
 
-        String labels = brokerConfig.getBrokerMetricsLabel();
+        String labels = brokerConfig.getMetricsLabel();
         if (StringUtils.isNotBlank(labels)) {
             List<String> kvPairs = Splitter.on(',').omitEmptyStrings().splitToList(labels);
             for (String item : kvPairs) {
                 String[] split = item.split(":");
                 if (split.length != 2) {
-                    LOGGER.warn("brokerMetricsLabel is not valid: {}", brokerConfig.getBrokerMetricsLabel());
+                    LOGGER.warn("metricsLabel is not valid: {}", brokerConfig.getMetricsLabel());
                     continue;
                 }
                 LABEL_MAP.put(split[0], split[1]);
             }
         }
-        if (brokerConfig.isBrokerMetricsPreferDelta()) {
+        if (brokerConfig.isMetricsPreferDelta()) {
```

Comment:
isMetricsPreferDelta --> isDeltaMetricPerferred

Reply:
I want to keep all metrics related config prefixed with "metrics". Do you have any better suggestions?

Reply:
> I want to keep all metrics related config prefixed with "metrics". Do you have any better suggestions?

How about isMetricInDelta?

Reply:
> How about isMetricInDelta?
I have done this modification.